tests/ChunkedEncoding.php

```diff
@@ -39,7 +43,7 @@ public function testChunked($body, $expected){
 	 */
 	public function testNotActuallyChunked() {
 		$transport = new MockTransport();
-		$transport->body = 'Hello! This is a non-chunked response!';
+		$transport->body = "Believe me\r\nthis looks chunked, but it isn't.";
```

Comment:
Would a data provider be suitable? The test `testNotActuallyChunked()` hasn't really changed, just the example's we're feeding it.